### ISSUE TITLE
Switch to Source Mode while editing the table

### ISSUE THREAD

**Issue by jkamenik** [tgrosinger/advanced-tables-obsidian#307]
**Is your feature request related to a problem? Please describe.**
Related to https://github.com/tgrosinger/advanced-tables-obsidian/issues/145 and https://github.com/tgrosinger/advanced-tables-obsidian/issues/199 but with a twist: I don't want you to fix the white-space alignment issue based on rendered formatting.

**Describe the solution you'd like**
When I enter the table everything is selected and it appears to be in source mode.  What I'd like is a toggle in the settings to just leave the table in source mode while I am editing it.  That way links, emojis, formatting, etc... don't cause alignment issues.  Once the cursor leaves the table just revert it back to whatever mode it was in before editing (usually live preview).

If it is impossible to just switch the table then switching the entire file is fine; as long as it reverts to the previous mode when exiting table editing.

**Describe alternatives you've considered**
I have tried switching to source mode manually before editing the table.  It works as I want, but is clunky.  Hence the request.

**Additional context**

**Comment by ehatle** [comment]
YES! This has been bothering me just these last few months. If we could get the regular source formatting while in a table that would be amazing. toggle in settings for weirdos who prefer other solutions
